operations-stdlib/src/commonMain/kotlin/string/Trim.kt

```diff
@@ -0,0 +1,61 @@
+package string
+
+import operation.StandardLogicOperation
+import utils.asList
+import utils.secondOrNull
+import utils.thirdOrNull
+
+object Trim : StandardLogicOperation, StringUnwrapStrategy {
+    override fun evaluateLogic(expression: Any?, data: Any?): Any?  {
+        return when (val firstItem = expression.asList.firstOrNull()) {
+            is String -> firstItem.trimElements(' ', TrimMode.BothEnds)
+            is List<*> -> {
+                val text = firstItem.firstOrNull()
+                val sign = firstItem.secondOrNull() ?: " "
+                val mode = (firstItem.thirdOrNull() as? String).toTrimMode() ?: TrimMode.BothEnds
+                (sign as? String)?.let {
+                    val char: Char = it.single()
+                    text.trimElements(char, mode)
+                }
+            }
+            else -> null
+        }
+    }
+
+    private fun String?.toTrimMode() = when (this) {
+        "start" -> TrimMode.Start
+        "end" -> TrimMode.End
+        "bothEnds" -> TrimMode.BothEnds
+        else -> null
+    }
+
+    private fun Any?.trimElements(char: Char, mode: TrimMode) =
```

Comment:
this fuction might be shortened to
<img width="747" alt="image" src="https://user-images.githubusercontent.com/84186160/172545488-8aff2c35-1c98-4ae2-9d52-fa990c0be56b.png">